cypress/integration/table/search.spec.ts

```diff
@@ -1213,8 +1215,10 @@ describe('Search', () => {
 
       cy.clearMocks();
 
-      cy.findByLabelText('from, date-time input').clear();
-      cy.findByLabelText('from, date-time input').type('2022-01-02 00:00');
+      // .clear doesn't work with datepickers in v6
+      // cy.findByLabelText('from, date-time input').clear();
```

Comment:
Again, this workaround may no longer be needed

Reply:
I've checked and `.clear` still doesn't work in cypress e2e tests.